src/System.Net.Security/ref/project.json

```diff
@@ -9,9 +9,9 @@
     "System.Threading.Tasks": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.6": {
```

Comment:
>Correct you cannot add more apis to netstandard16. Given this API doesn't exist anywhere else you actually should add it to only .NET Core similar to what @ericstj is doing in #11272.

@weshaggard We just found out that the docs are wrong and that we should have listed it as netstandard1.7.  Since that maps to .NET Framework 4.6.3 (vNext) *and* we plan to add this API there as well.